tests/views/test_api.py

```diff
@@ -34,18 +34,20 @@ class APIViewsTest(BaseTestCase):
     def setUp(self):
         self.app = self.create_app()
         self.client = self.app.test_client()
-        DB.create_all()
+        # only run the following few lines once
+        # DB.create_all()
+        # DB.session.add_all(self.starting_data)
+        # DB.session.commit()
 
     def tearDown(self):
-        DB.session.remove()
-        DB.drop_all()
+        # DB.session.close()
+        # DB.drop_all()
+        # DB.session.remove()
+        pass
 
     # **********  meta  *********
     def test_meta(self):
         """Is the meta object being generated?"""
-        DB.session.add_all(self.starting_data)
```

Comment:
What's the deal here? Will the tests run independently if a test adds or deletes a record?